doc/usermanual.md

```diff
@@ -83,7 +83,7 @@ Returns the current log level, in a json payload with a single attribute `level`
 
 ### Transactions
 The library implements a concept of transactions, in order to follow the execution flow the library follows when treating
-requests entering both from the Northbound and the Southbound.
+requests entering both from the North and the South ports of the IoT Agent.
 
 To follow the transactions, a new Domain is created for each incoming request; in the case of Northbound requests, this
```

Comment:
in the case of Northbound requests

Reply:
Fixed 376c3e2  Replaced with *requests received on the North Port of the IoT Agent*